cg/cli/demultiplex/finish.py

```diff
@@ -19,37 +19,32 @@ def finish_group():
     """Finish up after demultiplexing."""
 
 
-@finish_group.command(name="all")
+@finish_group.command(name="all-old-flow")
 @DRY_RUN
 @click.pass_obj
-def finish_all_cmd(context: CGConfig, dry_run: bool) -> None:
-    """Command to post-process all demultiplexed flow cells."""
+def finish_all_cmd_old_flow(context: CGConfig, dry_run: bool) -> None:
+    """Command to post-process all demultiplexed flow cells.
+    The old flow uses the DemuxPostProcessingNovaseqAPI.
+    """
     demux_post_processing_api: DemuxPostProcessingNovaseqAPI = DemuxPostProcessingNovaseqAPI(
         config=context
     )
     demux_post_processing_api.set_dry_run(dry_run=dry_run)
     demux_post_processing_api.finish_all_flow_cells()
 
-    # Temporary finish flow cell logic will replace logic above when validated
-    demux_post_processing_api_temp: DemuxPostProcessingAPI = DemuxPostProcessingAPI(config=context)
-    demux_post_processing_api_temp.set_dry_run(dry_run=dry_run)
-    is_error_raised: bool = demux_post_processing_api_temp.finish_all_flow_cells_temp()
-    if is_error_raised:
-        raise click.Abort
```

Comment:
In `demux_post_processing_api_temp.finish_all_flow_cells_temp()` we catch any error and return wether or not an error has occurred. Without the `raise click.Abort` in the end, we will not report any failures. I think a better solution can be found than the one we had, but we should exit with a failed exit code if an error has occured

Reply:
https://github.com/Clinical-Genomics/cg/pull/2389 addressed here!